src/api/DataHandler.ts

```diff
@@ -1,11 +1,61 @@
 import fetch from 'node-fetch';
 
-const IP = "http://localhost:8081" //"80.167.223.178";
+const api = "localhost:3306" //"80.167.223.178";
 
 enum HttpRequestType {
     Get,
     Post
 }
+//@ts-ignore
```

Comment:
TS-ignore?

Reply:
Written due to no usage of the API functions as there is no endpoint available as of yet.